src/OnePlusBot/Modules/Report.cs

```diff
@@ -12,6 +12,7 @@ namespace OnePlusBot.Modules
     public class ReportModule : ModuleBase<SocketCommandContext>
     {
         [Command("report")]
+        [Alias("r")]
```

Comment:
Same as other one letter aliases. I suggest `rep` but might be confused for "give reputation"